testsuite/libra-fuzzer/src/fuzz_targets/network_noise_stream.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::FuzzTargetImpl;
+use libra_proptest_helpers::ValueGenerator;
+use network::noise::fuzzing::{fuzz_post_handshake, generate_corpus};
+
+#[derive(Clone, Debug, Default)]
+pub struct NetworkNoiseStream;
+
+impl FuzzTargetImpl for NetworkNoiseStream {
+    fn name(&self) -> &'static str {
+        module_name!()
+    }
+
+    fn description(&self) -> &'static str {
+        "Network Noise crate stream"
+    }
+
+    fn generate(&self, _idx: usize, gen: &mut ValueGenerator) -> Option<Vec<u8>> {
+        Some(generate_corpus(gen))
```

Comment:
shouldn't the corpus be a valid message frame after handshake and not a handshake message?

Reply:
it shouldn't matter because the fuzzer will never generate a message that you can decrypt successfully